packages/paste-website/src/pages/components/side-modal/index.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: Side Modal
+package: '@twilio-paste/side-modal'
+description: An accessible modal dialog that sits on the right side of the page.
```

Comment:
suggestion: for descriptions here and on the package.json file we tend to follow the following pattern

> A(n) ${ComponentName} is a ...

"A Card is a styled container that groups related content and actions."
"A Popover is a page overlay triggered by a click that displays additional interactive content."
"A Modal is a page overlay that displays information and blocks interaction with the page until an action is taken or the Modal is dismissed."

Reply:
How is this?
> A Side Modal is a page overlay that displays information that does not block the user from interacting with the rest of the page.

Reply:
I buy that 👍🏼